requirements.txt

```diff
@@ -1,6 +1,7 @@
 coverage==4.4.1
 coveralls==1.1
 crayons==0.1.2
+distro
```

Comment:
Might be worth pinning it to 1.4.0 since that's the current release in https://pypi.org/project/distro/?

Reply:
It is better if the version is not fixed, as the distros are constantly evolving.

Normally should someboby continuous pinned Packages for a new version, test the new version und tell the provider if the package have a problem.